src/app/page/page.component.ts

```diff
@@ -126,6 +129,40 @@ export class PageComponent implements OnInit, OnDestroy {
     }
   }
 
+  setPageType(activePage): string {
+    if (
+      activePage instanceof
+      org.cru.godtools.shared.tool.parser.model.page.ContentPage
+    ) {
+      return 'content';
+    } else if (
+      activePage instanceof
+      org.cru.godtools.shared.tool.parser.model.page.CardCollectionPage
+    ) {
+      return 'card-collection';
+    } else {
+      return '';
+    }
+  }
+
+  setCardUrl = (card: number) => {
+    if (!this._pageParams?.langId) {
```

Comment:
I think that `this._pageParams` is always an object.

```suggestion
    if (!this._pageParams.langId) {
```